pages/productList.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FashiFy | Product List</title>
+
+    <link rel="stylesheet" href="../style.css" />
+  </head>
+  <body>
+    <div class="productlist-container">
+      <nav class="navigation">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="">
```

Comment:
One suggestion, instead of Blank href you can use "#" value

Reply:
It is fixed now.Thanks for your review